docs/reference/function-reference.md

```diff
@@ -349,29 +421,46 @@ The merged aggregation result
 
 #### Example
 
-```yaml
-functions:
-  merge_stats:
-    type: merger
-    code: |
-      # Merge two statistics objects
-      if value1 is None:
-        return value2
-      if value2 is None:
-        return value1
-
-      # Combine counts and sums
-      count = value1.get("count", 0) + value2.get("count", 0)
-      sum = value1.get("sum", 0) + value2.get("sum", 0)
-      result = {
-        "count": count,
-        "sum": sum,
-        "average": sum/count if count>0 else 0
-      }
+??? info "Producer - Session Activity Events (click to expand)"
 
-      return result
-    resultType: struct
-```
+    ```yaml
```

Comment:
Same here - if the user looks for a quick example on how the merger looks like 54 lines of producer code is overkill IMHO